### ISSUE TITLE
Tidy up user account page

### ISSUE THREAD

**Issue by kitenetter** [BiologicalRecordsCentre/iRecord#1176]
There are several items on the user account page that are no longer relevant, and that occasionally cause confusion for users. These should be removed from view:

Under 16 tick box
![image](https://user-images.githubusercontent.com/16039111/141492106-d677b784-dfd7-41f0-93fe-a8456c18e3be.png)

Redundant verification settings
![image](https://user-images.githubusercontent.com/16039111/141492175-f5ad29e0-a2d6-42c6-8583-f1c5c9d6e9ac.png)

Also, it would be preferable to have first name and last name adjacent to each other, rather than having the notification settings in the middle:

![image](https://user-images.githubusercontent.com/16039111/141492527-bc2e9a7e-17a9-4366-8a03-e4ada2286106.png)

**Comment by kitenetter** [comment]
Also, can we move the tick box for "Training mode" so that it is nearer the top.

**Comment by BirenRathod** [comment]
@kitenetter If we make any changes on current Drupal 7 version of iRecord, that will not impact anything on Drupal 9. Also, In a week time, we are going to stop using Drupal 7 version of iRecord, so I don't see any point of making changes there.

**Comment by kitenetter** [comment]
@BirenRathod agreed - it's not an urgent request, I just wanted to document the issue since it arose from a user enquiry today. Happy to wait until after the move to D9 has settled in.

**Comment by BirenRathod** [comment]
@kitenetter That is fine. I have amended that user profile page. But there are still some issues to resolve. That is for @johnvanbreda  to resolve. @johnvanbreda I have attached screenshot below.
1. Missing text in one of below option.
![image](https://user-images.githubusercontent.com/5941255/141783235-17037d52-fb24-4582-88f4-a6447a262586.png)

2. No details display in both below sections. Not sure if we decided to keep both sections or not.
![image](https://user-images.githubusercontent.com/5941255/141785825-184d125f-175c-426d-b1e2-722a011e402d.png)
![image](https://user-images.githubusercontent.com/5941255/141785900-096b1021-78f5-474b-b700-be4f3aec83b9.png)

For info ... on Drupal 9 iRecord, I'm not using "terms of use" module any longer. I think @johnvanbreda mentioned while ago that "Legal" module is provide better features, So I have enabled that now iRecord Drupal 9.

**Comment by DavidHepper** [comment]
Please also change the checkbox prompt 'Training' to 'Training Mode', if not already done.

**Comment by johnvanbreda** [comment]
I've fixes the empty "Georeferencing services override" option.

A few things:
* The System section of the form should be hidden.
* "where you record" needs a leading capital.
* The terms and conditions box is a bit short.
* My recording experience - the box says it is disabled as I don't have permission to edit it.

**Comment by BirenRathod** [comment]
@johnvanbreda 
- System section now hidden
- "where you record" has now got leading capital.
- The terms and condition box is not possible to increase in size as it is control by "Legal" module.
- "My recording experience" will be enabled when I do final migration 24th.
@DavidHepper I have changed "Training" to "Training Mode" now.

**Comment by johnvanbreda** [comment]
@BirenRathod the terms and conditions box has a class `legal-terms-scroll` which has a height set to 20em. So you could add something like the following to the theme's CSS:
```css
div.legal-terms-scroll {
  height: 50em;
}
```

**Comment by BirenRathod** [comment]
@johnvanbreda Thanks for that, I will add it. Did you look at other two things? why "Species groups" & "Recording locations" don't display?

**Comment by johnvanbreda** [comment]
@BirenRathod the Drupal 8/9 version of Easy Login does not have the fields added to the user profile (field_location and field_location_expertise, also field_taxon_groups and field_taxon_groups_expertise). The expertise fields were originally defined to store the user's verification limit but it is no longer done that way so is no longer needed. A while ago there was a discussion as to whether to continue storing the users' "recording location" and species groups in the profile but I don't think it was concluded, as one other idea was to use the user's saved filters instead.

So, the fields that are currently in the data are probably just a result of your migration, not added by the Easy Login module. 

@DavidRoy @kitenetter we need to decide whether to continue with the current storing of preferred vice county and taxon groups in the user profile and if so, there is a bit of code to migrate which either Biren or I will need to be allocated to.

**Comment by BirenRathod** [comment]
@johnvanbreda for this details. 
@johnvanbreda , @kitenetter & @DavidRoy 
If we decided not to use those fields then I will delete them from user profile. It will be less work of migrating those data from Drupal 7 too.

**Comment by johnvanbreda** [comment]
The location and taxon groups aspect of the user profile has now been coded and the configuration updated, so they should both now be populated with the correct list of options.

**Comment by BirenRathod** [comment]
@kitenetter This has resolved now.

**Comment by kitenetter** [comment]
Remaining issues:

1. First name and last name are still separated, with Google Analytics box dividing them:
![image](https://user-images.githubusercontent.com/16039111/144672879-355d8f9d-1f50-4953-a66a-58bf281cabce.png)

2. It would be good to have some sort of drop-down arrow next to the headings for what and where you record, to indicate that these are clickable subheadings:
![image](https://user-images.githubusercontent.com/16039111/144672958-f50f7ef6-9e82-4264-b12d-77b6f34d066a.png)

3. The Training Mode tick box is very far down the page - I would prefer it to be shown underneath the two password boxes.

**Comment by DavidHepper** [comment]
@kitenetter - agree with the above, esp. the location of Training Mode.
While you're at it, I notice that after changing (set or clear) the Training Mode flag and clicking Save, the page refreshes (as expected) but the banner warning of being in Training Mode is shown based on the _previous_ setting of the flag, not on the new status just saved. This is confusing. It only corrects itself if the Account page is then refreshed manually.

**Comment by BirenRathod** [comment]
@kitenetter Please close this if you happy with user display page.
